test/qweb3.js

```diff
@@ -4,17 +4,24 @@ const chai = require('chai');
 const assert = chai.assert;
 const expect = chai.expect;
 
+const bs58 = require('bs58')
+
 const Qweb3 = require('../src/qweb3');
 const Formatter = require('../src/formatter');
 const Config = require('./config/config');
 const ContractMetadata = require('./data/contract_metadata');
 
+// Load some environment variables as global vars
+require('dotenv').config();
+var qtumRPCAddress = "QTUM_TESTNET_RPC_ADDRESS" in process.env ? String(new Buffer(process.env["QTUM_TESTNET_RPC_ADDRESS"])) : console.debug('QTUM_TESTNET_RPC_ADDRESS environment variable not found')
+var qtumAddress = "QTUM_TESTNET_ADDRESS" in process.env ? String(new Buffer(process.env["QTUM_TESTNET_ADDRESS"])) : console.debug('QTUM_TESTNET_ADDRESS environment variable not found')
```

Comment:
lets create two exported functions in `test/config/config.js` so this can be used in multiple test classes.
`getQtumRPCAddress()` & `getDefaultQtumAddress()`

Reply:
Got it! It seems the best approach to improve it.
I'll work on it.